overlord/ifacestate/ifacemgr.go

```diff
@@ -174,6 +192,17 @@ func (m *InterfaceManager) StartUp() error {
 	if _, err := m.reloadConnections(""); err != nil {
 		return err
 	}
+	if useAppArmorPrompting(m) {
+		if err := m.initInterfacesRequestsManager(); err != nil {
+			// TODO: if this fails, set useAppArmorPromptingValue to false ?
+			// If this is done before profilesNeedRegeneration, then profiles will only
+			// be regenerated if prompting is newly supported and the backends were
+			// successfully created.
+			// TODO: also set "apparmor-prompting" flag to false?
+			// XXX: should this log an error, rather than returning error?
```

Comment:
I think yes, that this should only log as long as continuing does reasonable things